<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MediaFileView" name="tvMediaFiles">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="leSearchText">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionScan_media_library"/>
    <addaction name="actionRead_media_data"/>
    <addaction name="actionWrite_media_data"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionMedia_Libraries"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="sbMain"/>
  <action name="actionScan_media_library">
   <property name="text">
    <string>&amp;Scan media library</string>
   </property>
  </action>
  <action name="actionRead_media_data">
   <property name="text">
    <string>&amp;Read media data</string>
   </property>
   <property name="toolTip">
    <string>Read media data from XML-file</string>
   </property>
  </action>
  <action name="actionWrite_media_data">
   <property name="text">
    <string>&amp;Write media data</string>
   </property>
  </action>
  <action name="actionMedia_Libraries">
   <property name="text">
    <string>Media &amp;Libraries...</string>
   </property>
  </action>
  <action name="actionEdit_Settings">
   <property name="text">
    <string>Edit &amp;Settings</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MediaFileView</class>
   <extends>QTableView</extends>
   <header>mediafileview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>playFile()</slot>
  <slot>scanMediaLibrary()</slot>
 </slots>
</ui>
